src/lib/auth.js

```diff
@@ -7,10 +7,14 @@ const { disableAuth } = require('./../config')
 // `secured` will be rejected without these headers.
 const secured = function secured(req, res, next) {
   if ((req.session && req.isAuthenticated()) || disableAuth) {
+    delete req.session.authBlockedRedirectUrl
```

Comment:
I'm not 100% familiar with the session vars available - will `authBlockedRedirectUrl` always be present for authenticated sessions? Will this `delete` complain if it's not there?

Reply:
it will be cleared out if a session is ever actually authenticated, so it will only be available once. The only time it's therefore stored is between the `/auth` route being blocked and the `/callback` route being redirected back from GitHub.

`delete` in JS is a bit of the wild west, it won't complain so long as what it's deleting from (req.session) exists, the conditional above should guarantee that